### ISSUE TITLE
Build docker image with Alpine Linux

### ISSUE THREAD

**Issue by msokk** [msokk/electron-render-service#10]
- [ ] Build Electron on Alpine
- [ ] Submit the build script to aports

**Comment by ugin** [comment]
Have you tried to build electron on Alpine? I've tried to build it on CentOS and had a lot of issues.

**Comment by msokk** [comment]
I haven't yet.
I know that it is a quite a bit of work because of musl C library and gconf/dconf differences.

Atleast there is a port of Chromium in Alpine - http://git.alpinelinux.org/cgit/aports/tree/community/chromium

**Comment by burningTyger** [comment]
I'd wait and see how others fare with this idea. There are issues all around compiling electron on alpine :)

**Comment by msokk** [comment]
Yup, not easy, I tried to build electron on Alpine half a year ago, got stuck with gconf 2 missing, but looks like libchromiumcontent can be compiled without gconf - https://github.com/electron/electron/issues/2727
